entities/BizAI/AIChats.json

```diff
@@ -0,0 +1,63 @@
+{
+  "type": "object",
+  "properties": {
+    "success": {
+      "type": "boolean"
+    },
```

Comment:
we dont need the response object in the entity. please remove it

Reply:
GPT 😅